CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+project(asio-rawsockets-example)
+cmake_minimum_required(VERSION 3.2)
+
+add_compile_options(-std=c++11)
+
+find_package(Boost REQUIRED system)
+
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIR})
```

Comment:
Can you use `target_include_directories()` instead? Even though this file may not ever have other targets, I think it'd keep us consistent with your usage of `target_link_libraries()` below (i.e. instead of `link_directories()`), and it also lets us more fine-grained control over the scope.

```suggestion
```